content/klc/S2.1.adoc

```diff
@@ -4,11 +4,12 @@ brief = "General symbol naming guidelines"
 
 . Library naming should not be duplicated in footprint name
 . If symbol with same name exists for multiple manufacturers, the manufacturer name is written first
-. Specific manufacturer name (for fully specified symbols)
-. Type of symbol (for generic symbols)
+. Fully specified symbols are named based on the manufacturer part number
+.. If multiple manufacturers build compatible devices then the common base part number is used with clear name suffix for the footprint (e.g. `L78L05_TO92`)
```

Comment:
Perhaps change "compatible" to "interchangeable" so show an exact equivalence?

Perhaps change "used with clear name suffix for the footprint" with "suffixed with the footprint type"?

Reply:
No if one manufacturer builds a device then we use the full part number. The suffix is a workaround for manufacturers using different part numbers for the same exact device.

Reply:
Ah, I see. Perhaps make that explicit? e.g. `.. Since the full MPN already implies the footprint of the part, there is no need to add it to the name. But if multiple manufacturers build compatible devices (but with different designators for the footprint) then the common base part number is used with clear name suffix for the footprint (e.g. `L78L05_TO92`)`

Reply:
That is way too long for this set of rules. Really a bit of common sense is to be expected by users. i will simply replace the if with if and only if (short iff)